src/p5xr/core/p5xrviewer.js

```diff
@@ -48,3 +54,19 @@ p5.prototype.setViewerPosition = function(x, y, z) {
   let viewer = p5xr.instance.viewer;
   viewer.setPosition(x, y, z);
 };
+
+p5.prototype.sticky = function(drawOnTop = false) {
+  push();
+  p5xr.instance.viewer.drawOnTop = drawOnTop;
+  if(drawOnTop)
```

Comment:
I am not sure if I agree with keeping this as a property on the viewer. Seems a bit odd to have a `drawOnTop` property on a viewer. The conceptual link between a viewer and the order of rendering is a little tenuous. There is very little cost to enabling DEPTH_TEST with `noSticky` every time it is called.